Modix.Services/MessageLogging/MessageLoggingLogMessages.cs

```diff
@@ -0,0 +1,209 @@
+                "GuildId: {GuildId}\r\n\tChannelId: {ChannelId}\r\n\tMessageId: {MessageId}");
+
+        public static void IgnoringNonGuildMessage(
+                ILogger logger)
+            => _ignoringNonGuildMessage.Invoke(
+                logger);
+        private static readonly Action<ILogger> _ignoringNonGuildMessage
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.IgnoringNonGuildMessage.ToEventId(),
+                    "Ignoring Message: Does not belong to a Guild")
+                .WithoutException();
+
+        public static void IgnoringSelfAuthoredMessage(
+                ILogger logger)
+            => _ignoringSelfAuthoredMessage.Invoke(
+                logger);
+        private static readonly Action<ILogger> _ignoringSelfAuthoredMessage
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.IgnoringSelfAuthoredMessage.ToEventId(),
+                    "Ignoring Message: Author is Self")
+                .WithoutException();
+
+        public static void IgnoringUnchangedMessage(
+                ILogger logger)
+            => _ignoringUnchangedMessage.Invoke(
+                logger);
+        private static readonly Action<ILogger> _ignoringUnchangedMessage
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.IgnoringUnchangedMessage.ToEventId(),
+                    "Ignoring Message: Content is unchanged")
+                .WithoutException();
+
+        public static void IgnoringUnmoderatedChannel(
+                ILogger logger)
+            => _ignoringUnmoderatedChannel.Invoke(
+                logger);
+        private static readonly Action<ILogger> _ignoringUnmoderatedChannel
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.IgnoringUnmoderatedChannel.ToEventId(),
+                    "Ignoring Message: Channel is Unmoderated")
+                .WithoutException();
+
+        public static void MessageDeletedHandled(
+                ILogger logger)
+            => _messageDeletedHandled.Invoke(
+                logger);
+        private static readonly Action<ILogger> _messageDeletedHandled
+            = LoggerMessage.Define(
+                    LogLevel.Information,
+                    EventType.MessageDeletedHandled.ToEventId(),
+                    "Handling deleted message")
+                .WithoutException();
+
+        public static void MessageDeletedHandling(
+                ILogger logger)
+            => _messageDeletedHandling.Invoke(
+                logger);
+        private static readonly Action<ILogger> _messageDeletedHandling
+            = LoggerMessage.Define(
+                    LogLevel.Information,
+                    EventType.MessageDeletedHandling.ToEventId(),
+                    "Deleted message handled")
+                .WithoutException();
```

Comment:
Is the text for these messages swapped? `MessageDeletedHandled` says "handling" and `MessageDeletedHandling` says "handled"

Reply:
Yup.

Reply:
(also applies to the update ones below)

Reply:
Goddamnit